docs/DeveloperGuide.md

```diff
@@ -196,6 +192,15 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 This section describes some noteworthy details on how certain features are implemented.
 
 The features mentioned are:
+<!---
+- [Adding contacts with optional arguments](#add-contacts-with-optional-arguments)
+- [Finding contacts by keywords](#find-contacts-by-keywords)
+- [Deleting multiple contacts by keywords](#delete-by-keywords)
+- [Importing a JSON file](#import-json-file)
+- [Support for multiple social handles](#support-for-multiple-social-handles)
+- [Undoing / redoing command](#proposed-undoredo-feature)
+- [Input Suggestion](#input-suggestion)
+--->
```

Comment:
Can I ask what is this for?

Reply:
This is the commented out section of the previous version. I will remove it.